aws-s3/latest/config/direct-access.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Direct Access URLs
+---
+
+The main purpose of Direct Access URLs (or DAUs) is to accelerate the local download of content by allowing you to have direct content URLs for your binary content that can help with distributed content repositories in customer environments, and cloud deployments.
```

Comment:
What do we actually mean by "with distributed content repositories in customer environments" - otherwise remove for now ?